From 86de78e49c2872c0a8dccd915f48d626f88c7b1d Mon Sep 17 00:00:00 2001
From: Benjamin Berg <bberg@redhat.com>
Date: Mon, 2 Sep 2019 11:39:03 +0200
Subject: [PATCH 3/3] xsettings: Ensure plugin is started after the shell

In the processes of adding support to auto-shutdown Xwayland, the
"After" rule was accidentally deleted. Re-add the rule, as otherwise
gsd-xsettings may be started before gnome-shell is and that causes
an immediate session failure.

Fixes: #442
---
 plugins/xsettings/gsd-xsettings.service.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/plugins/xsettings/gsd-xsettings.service.in b/plugins/xsettings/gsd-xsettings.service.in
index d91c2fa4..36527f20 100644
--- a/plugins/xsettings/gsd-xsettings.service.in
+++ b/plugins/xsettings/gsd-xsettings.service.in
@@ -9,6 +9,7 @@ RefuseManualStart=true
 RefuseManualStop=true
 
 PartOf=gsd-xsettings.target
+After=gnome-session-initialized.target
 
 [Service]
 Type=dbus
-- 
2.20.1

